src/dev/konnector.md

```diff
@@ -508,6 +508,33 @@ Cozy will soon have a store for connectors and you will be able to publish conne
 at the moment, you need to declare your new connector on the [cozy forum](https://forum.cozy.io).
 The Cozy team will review your code and add your connector to the [Cozy Collect] application.
 
+To make the connector available more quickly for all cozys, you can follow this few steps of 
+pre-packaging:
```

Comment:
Why is it called "__pre__-packaging" instead of "packaging"?

Reply:
Because in the 'packaging' we also include PR to collect, Sentry, renovate and more.

Reply:
Ok, but as we do not speak about this part, contributors will not understand where is the _pre-packaging_ and where is the _packaging_, imho.

Reply:
It is like you speak about _part 1_ and never speak about _part 2_, the contributor's brain seeks the _part 2_ because it is a logical sequence.